src/NServiceBus.Core/Hosting/Helpers/AssemblyScannerConfiguration.cs

```diff
@@ -24,7 +24,7 @@ public class AssemblyScannerConfiguration
         /// <summary>
         /// Defines whether nested directories should be included in the assembly scanning process. Default value is <code>false</code>.
         /// </summary>
-        public bool ScanAssembliesInNestedDirectories { get; set; } = false;
+        public bool ScanAssembliesInNestedDirectories { get; set; }
```

Comment:
`ScanAppDomainAssemblies` is also being set to `false`, so seems like that should be done too, if default values are going to be relied on.